TF_Settings_Web/src/Pages/About/About.module.scss

```diff
@@ -0,0 +1,91 @@
+@use '@/variables.module.scss' as v;
+@use '@/App.module.scss';
+
+.title-line {
+    @include App.title-line;
+}
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 1vh 0;
+}
+
+.container {
+    margin: 0 2vw;
+}
+
+.page-divider {
+    @include App.page-divider;
+}
+
+.info-table {
+    $numCols: 1;
```

Comment:
🙈 This is never used

Reply:
Leftover from trying to do something clever with nth-last-child(-n + $nulCols) but it doesn't accept variables - will remove